README.md

```diff
@@ -29,6 +29,13 @@ PhysicsTutorials.open_notebooks()
 Supported source files for tutorials are Jupyter notebooks, Weave.jl files, or Literate.jl files.
 To contribute a tutorial, clone the repository and put the source file into `tutorials/<category>/<tutorial_name>/` and name it `<tutorial_name>.ipynb` (extension `.jmd`/`.jl` for Weave/Literate sources). To trigger the generation process of all output formats, run the following code from within the repository root folder:
```

Comment:
We should change the "to trigger the generation process of all ...." by "to trigger the conversion to all other output formats...".